freqtrade/persistence/models.py

```diff
@@ -257,6 +257,7 @@ class LocalTrade():
     sell_reason: str = ''
     sell_order_status: str = ''
     strategy: str = ''
+    buy_tag: Optional[str] = None
```

Comment:
You might wanna add this to the `to_json()` of this method, too - so it's available via API as well.

Note:
to have it available via API, you'll ALSO need to add it to the proper `api_schema.py` schema, otherwise it'll be hidden from the API.

Reply:
done 🙏 anything else for this PR?